### PR TITLE
question list redo 2

### PR DESCRIPTION
**Pull request by SBrandon0432**

### CODE REVIEW

**Review by SBrandon0432 — COMMENTED** [review]
looks good

File: src/Components/QuestionList/QuestionList.tsx [hunk 1]
```diff
@@ -51,31 +41,16 @@ export const QuestionList = () => {
...
+            <EditOptions
+                filterSettings={filterSettings}
+                setFilterSettings={setFilterSettings}
+                setShowDelete={setShowDelete}
+                showDelete={showDelete}
+            />
+            <QuestionCards
+                currentQuestions={currentQuestions}
+                showDelete={showDelete}
+            />
```
Comment (CommanderHamster) [inline src/Components/QuestionList/QuestionList.tsx:53]: This is what we were trying to rewrite

File: src/Components/Utils/Utils.tsx [hunk 2]
```diff
@@ -23,20 +23,18 @@ export const isQuestionHideCorrect = (
     }
 };
 
-export const updateData = (
-    updatedQuestion: Question,
-    setQuestions: any,
-    questions: Question[]
+export const filterQuestions = (
```
Comment (CommanderHamster) [inline src/Components/Utils/Utils.tsx:26]: This is better than the first implementation

File: src/Components/QuestionList/QuestionList.tsx [hunk 3]
```diff
@@ -20,26 +21,15 @@ import {
...
-        (question: Question, index: number) => {
-            return (
-                isQuestionType(question, type as Tags) &&
-                isQuestionName(question, name) &&
-                isQuestionDifficulty(question, difficulty as Difficulty) &&
-                isQuestionHideCorrect(question, !!hideCorrect)
-            );
-        }
-    );
+    const currentQuestions = filterQuestions(questions, filterSettings);
```
Comment (CommanderHamster) [inline src/Components/QuestionList/QuestionList.tsx:32]: 👍

**Pull request closed without merging**